BackofficeBundle/Resources/public/coffee/AdminFormView.coffee

```diff
@@ -32,20 +32,20 @@ AdminFormView = OrchestraView.extend(
       url: @options.url
       method: @method
       success: (response) ->
+        originalButton = $('.submit_form', response)
+        button = originalButton.clone().attr('data-clone', originalButton.attr('id')).removeAttr('id')
+        actionButtons = $('<div>')
         if viewContext.deleteButton
-          originalButton = $('.submit_form', response)
-          button = originalButton.clone().attr('data-clone', originalButton.attr('id')).removeAttr('id')
-          footer = $('<div>')
-          .append(viewContext.renderTemplate('OpenOrchestraBackofficeBundle:BackOffice:Underscore/deleteButton', viewContext.options))
-          .prepend(button)
-          .html()
+          actionButtons.append(viewContext.renderTemplate('OpenOrchestraBackofficeBundle:BackOffice:Underscore/deleteButton', viewContext.options))
+        actionButtons = actionButtons.prepend(button)
+        .html()
```

Comment:
you can remove the line return before ``.html()``, not sure it helps with the reading